prql-compiler/tests/integration-vendors/docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  postgres:
+    image: "postgres:15-alpine"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: dummy
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: root
+  mysql:
+    image: "mysql:8"
+    ports:
+      - "3306:3306"
+    environment:
+      MYSQL_DATABASE: dummy
+      MYSQL_ROOT_PASSWORD: root
+  #  db2:
+  #    image: 'icr.io/db2_community/db2'
+  #    ports:
+  #      - '50000:50000'
+  #    environment:
+  #      LICENSE: accept
+  #      DBNAME: dummy
+  #      DB2INSTANCE: db2
+  #      DB2INST1_PASSWORD: root
+  #      BLU: false
+  #      TO_CREATE_SAMPLEDB: false
+  #      REPODB: false
+  #      IS_OSXFS: false
+  mssql:
+    image: "mcr.microsoft.com/mssql/server:2022-latest"
```

```suggestion
    image: "mcr.microsoft.com/mssql/server:latest"
```

It does not seem necessary to specify the version.

```sh
$ docker buildx imagetools inspect mcr.microsoft.com/mssql/server:2022-latest
Name:      mcr.microsoft.com/mssql/server:2022-latest
MediaType: application/vnd.docker.distribution.manifest.v2+json
Digest:    sha256:7dfe6bf43d18d0cf929e1dc40d0c1fccce02e7cca1ed9e277e35b6815108d17b

$ docker buildx imagetools inspect mcr.microsoft.com/mssql/server:latest
Name:      mcr.microsoft.com/mssql/server:latest
MediaType: application/vnd.docker.distribution.manifest.v2+json
Digest:    sha256:7dfe6bf43d18d0cf929e1dc40d0c1fccce02e7cca1ed9e277e35b6815108d17b
```